src/Components/Authentication/Desktop/ModalManager.tsx

```diff
@@ -67,7 +74,18 @@ export class ModalManager extends Component<
         onClose={this.closeModal}
         subtitle={copy}
       >
-        <FormSwitcher type={currentType} handleSubmit={handleSubmit} />
+        <FormSwitcher
+          type={currentType}
+          handleSubmit={handleSubmit}
+          onFacebookLogin={() =>
+            (window.location.href =
+              submitUrls.facebook + "?redirect-to=" + redirectUrl)
+          }
+          onTwitterLogin={() =>
+            (window.location.href =
+              submitUrls.twitter + "?redirect-to=" + redirectUrl)
```

Comment:
Do we need to update this to include the other params like signup-intent and signup-referrer?

Reply:
Yeah, we need to pass in other params as well